src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -38,7 +49,10 @@ const ArtworkDetails = loadable(
       /* webpackChunkName: "consignBundle" */ "./Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails"
     ),
   {
-    resolveComponent: component => component.ArtworkDetails,
+    resolveComponent: (component, props: any) => {
+      props.match.location.query = consignUtmMarks
```

Comment:
We should avoid mutating the `props` object like this

Reply:
yeah, I didn't like it either. Fixed